webapp/src/containers/addOrEditSubscriptions/subComponents/index.tsx

```diff
@@ -168,15 +169,26 @@ const AddOrEditSubscription = ({open, close, subscriptionData}: AddOrEditSubscri
     useApiRequestCompletionState({
         serviceName: Constants.pluginApiServiceConfigs.checkSubscriptionsConfigured.apiServiceName,
         handleError: (error) => {
-            dispatch(resetGlobalModalState());
-            if (
-            error?.id !== Constants.ApiErrorIdSubscriptionsNotConfigured &&
-            error?.id !== Constants.ApiErrorIdSubscriptionsUnauthorized &&
-            error?.id !== Constants.ApiErrorIdNotConnected
-            ) {
-                return handleErrorComponent(error);
+            if (error) {
+                if (error?.id === Constants.ApiErrorIdRefreshTokenExpired) {
```

Comment:
```suggestion
                if (error.id === Constants.ApiErrorIdRefreshTokenExpired || error.id === Constants.ApiErrorIdNotConnected) {
```
Remove `?` from everywhere as you've already applied the `if` condition above.